layouts/partials/hooks/body-end.html

```diff
@@ -23,6 +23,7 @@
       document.body.appendChild(target);
     }
     target.value = document.getElementById(elem).innerText;
+    target.value = target.value.replace(/\n\n/gm,"\n").trim();
```

Comment:
Thanks @ayushtamra 

I think this may be the wrong fix. If the original bug / behavior changes within Hugo then what gets copied could be different from what you get when you download the example.

At a minimum, this needs a clear comment that explains _why_ the `replace()` happens.

Reply:
Thanks for the feedback @sftim I am a bit not clear with what you mentioned. Though I would like to mention that maybe its related to the `execCommand()` which is doing the copy here.
Also I have seen an question posted on StackOverflow(don't know if it makes sense): https://stackoverflow.com/questions/34477425/javascript-execcommandcopy-copies-text-but-adds-extra-white-space-to-value
But I am sure to look into and learn if there is any issues related with hugo